graphene_django/rest_framework/mutation.py

```diff
@@ -69,7 +69,7 @@ def __init_subclass_with_meta__(cls, serializer_class=None,
 
     @classmethod
     def mutate_and_get_payload(cls, root, info, **input):
-        serializer = cls._meta.serializer_class(data=input)
+        serializer = cls._meta.serializer_class(data=input, context=getattr(info, 'context', None))
```

Comment:
I believe there is a chance this might break if someone has overridden their serializer class's `__init__` in such a way as to not accept `**kwargs`. Even then, if they don't pop the `context` kwarg off before the base DRF `Field` class is resolved, I think this could cause an error.